pkgs/tools/audio/beets/default.nix

```diff
@@ -76,5 +76,6 @@ lib.makeExtensible (
   }
   // lib.optionalAttrs config.allowAliases {
```

Comment:
I believe this should not be gated under `config.allowAliases` and instead be put in the upper attribute set. That will probably also make ofBorg find it.